pkg/snapshot/restorer/restorer.go

```diff
@@ -290,11 +295,11 @@ func makeWALAndSnap(waldir, snapdir string, cl *membership.RaftCluster, restoreN
 			Index: commit,
 			Term:  term,
 			ConfState: raftpb.ConfState{
-				Nodes: nodeIDs,
+				Voters: nodeIDs,
```

Comment:
Does this get serialized in the snapshot? If so, we might have a backward compatibility problem.

Reply:
No Amshu, it does not. See [here](https://github.com/etcd-io/etcd/blob/7bd956fa2bf963abfd33eac084ee90ffdf2003c7/raft/raftpb/raft.pb.go#L664). From an API point of view, it's just a naming change as mentioned in the [changelog](https://github.com/etcd-io/etcd/blob/master/CHANGELOG-3.4.md#package-raft).